addon_service/authorized_storage_account/serializers.py

```diff
@@ -112,6 +112,26 @@ def create(self, validated_data):
 
         return authorized_account
 
+    def update(self, instance, validated_data):
```

Comment:
Do we want to do the work to complain if we're given values to update that aren't supported?

Reply:
(this is an actual question, not midwestern passive aggression)

Reply:
hmm could do... tho unfortunately, ember-data doesn't (iirc) have a sense of "read-only" or "write-once-on-create" attributes, so all fields are included on every PATCH -- could add frontend work to avoid that, but seems easier to ignore non-updatable fields (leaning toward permissiveness in what we accept)

Reply:
Yeah, we would need to compare all non-updatable values to what is currently set if we wanted to have explicitly "non-updatable" fields.